### PR TITLE
Add /users/{username}/activity/current to get current activity

### PR DESCRIPTION
**Pull request by lajp**
This would be a cool addition that people could use for example in their dashboards and status bars

### CODE REVIEW

File: src/api/users.rs [hunk 1]
```diff
@@ -56,6 +57,51 @@ pub async fn delete_user(
...
+            || block(move || conn.are_friends(user.id, target_user.id)).await??
+        {
+            target_user.id
+        } else {
+            return Err(TimeError::Unauthorized);
+        }
+    };
+
+    match heartbeats.get(&target_user) {
+        Some(a) => {
```
Comment (DrVilepis) [inline src/api/users.rs:91]: Please fix the variable names, what the hell does `a` or `rs` mean

`started` is confusing too

**Review by DrVilepis — CHANGES_REQUESTED** [review]
Please make code readable

File: src/api/users.rs (hunk 1 above)
Comment (bluelhf) [reply src/api/users.rs:91]: Perhaps `heartbeat`, `last_heartbeat`, and `session_start` instead of `a`, `heartbeat`, and `started`, respectively?

Comment (DrVilepis) [reply src/api/users.rs:91]: Seems good @lajp godo

**Comment by lajp** [comment]
@DrVilepis good enough?

**Comment by DrVilepis** [comment]
yes this is good

**Pull request merged**